src/Utility/TextColor.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5\Utility;
+
+/**
+ * Text colors.
+ *
+ * @see https://getbootstrap.com/docs/5.3/utilities/colors/#colors
+ */
+enum TextColor: string
```

Comment:
`text-black-50` and `text-white-50` are missing

Reply:
![image](https://github.com/user-attachments/assets/e1fed11a-6095-4a2b-8ce9-4d8add76aee8)